packages/frontend/src/components/composer/ComposerMessageInput.tsx

```diff
@@ -118,6 +118,10 @@ export default class ComposerMessageInput extends React.Component<
     return this.state.text
   }
 
+  hasText(): boolean {
+    return !this.getText().match(/^\s*$/)
```

Comment:
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/trim and then checking length might be easier to understand code?

Reply:
I just moved the existing regex, so the regex was not introduced with this PR